packages/iov-ethereum/src/ethereumcodec.ts

```diff
@@ -0,0 +1,51 @@
+// import createKeccakHash from "keccak";
+
+import {
+  Address,
+  Nonce,
+  SignedTransaction,
+  SigningJob,
+  TransactionIdBytes,
+  TxCodec,
+  UnsignedTransaction,
+} from "@iov/bcp-types";
+import { Keccak256 } from "@iov/crypto";
+import { Encoding } from "@iov/encoding";
+import { ChainId, PostableBytes, PublicKeyBundle } from "@iov/tendermint-types";
+
+const { toHex } = Encoding;
+
+export function toChecksumAddress(address: string): Address {
+  // https://github.com/ethereum/EIPs/blob/master/EIPS/eip-55.md
+  const addressLower = address.toLowerCase().replace("0x", "") as Address;
+  const addressHash = toHex(new Keccak256(addressLower).digest());
+  let checksumAddress = "0x";
+  for (let i = 0; i < 40; i++) {
+    checksumAddress += parseInt(addressHash[i], 16) > 7 ? addressLower[i].toUpperCase() : addressLower[i];
+  }
+  return checksumAddress as Address;
+}
+
+export const ethereumCodec: TxCodec = {
```

Comment:
can you export `ethereumCodec` in index.ts please? Then it can be tested from the cli